train.py

```diff
@@ -1,11 +1,107 @@
 import hydra
+from attr import dataclass
```

Comment:
that's odd.
If you want to use attr, you can use it directly (OmegaConf supports attr classes).
If you want to use dataclass, you should import it from dataclasses (3.7+, there is a dataclasses pip package for 3.6).
```
from dataclasses import dataclass
```